ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -59,14 +60,24 @@ define([
             var loginUser = user.get('user');
             if(loginUser.attributes.username == this.model.attributes.createdBy)
             {
-                $.ajax({
-                    type: 'Delete',
-                    context: this,
-                    url: '/search/catalog/internal/forms/' + this.model.id,
-                    contentType: 'application/json',
-                });;
-                var index = this.model;
-                this.model.collection.remove(index);
+                this.listenTo(ConfirmationView.generateConfirmation({
+                    prompt: 'This will permanently delete the template. Are you sure? ',
+                    no: 'Cancel',
+                    yes: 'Delete'
+                }),
+                'change:choice',
+                function(confirmation) {
```

Comment:
This cant get reduced to `this.model.destroy()` in this instance by attaching the `/search/catalog/internal/forms/` url to the model collection.

Reply:
Updated to uses backbone solutions